src/main/webapp/scripts/user-script.js

```diff
@@ -457,11 +457,28 @@ function triggerDelete(e){
     })
 }
 
+// trigger warning if password is too short
+function shortPassword(){
+    var password = document.getElementById("add-password").value;
+    var passwordWarning = document.getElementById("password-warning");
+    if(password.length < 8){
```

Comment:
Is this check needed? Won't the api return an error if the password doesn't meet the requirements?

Reply:
The API does return an error, but I want users to be aware of this when they are typing password to save clicks.